benchalerts/parse_conbench.py

```diff
@@ -94,15 +86,15 @@ def regression_summary(
     return summary
 
 
-def regression_details(all_comparisons: dict) -> str:
+def regression_details(comparisons: List[ConbenchClient.RunComparison]) -> str:
```

Comment:
If this lives outside `ConbenchClient` it seems like `RunComparison` probably should too. And maybe in a different file.